README.md

```diff
@@ -138,6 +138,83 @@ rstring -i
 
 5. **Large projects may produce substantial output**: Use `--preview-length` or specific patterns for better manageability.
 
+## Development
+
+If you'd like to contribute to Rstring or set it up for local development, follow these steps:
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/tnunamak/rstring.git
+   cd rstring
+   ```
+
+2. Create a virtual environment and activate it:
+   ```bash
+   python -m venv venv
+   source venv/bin/activate  # On Windows, use `venv\Scripts\activate`
+   ```
+
+3. Install the development dependencies:
+   ```bash
+   pip install -r requirements-dev.txt
+   ```
+
+4. Install the package in editable mode:
+   ```bash
+   pip install -e .
+   ```
+
+5. Run the tests:
+   ```bash
+   pytest
+   ```
+
+### Running Rstring Locally
+
+To run Rstring locally during development:
+
+1. Make sure you're in the project root directory and your virtual environment is activated.
+
+2. Run Rstring using the Python interpreter:
+   ```bash
+   python -m rstring [options]
+   ```
+
+   For example:
+   ```bash
+   python -m rstring --include=*/ --include=*.py --exclude=*
+   ```
+
+3. If you want to use the `rstring` command directly, you can create an alias or a batch file:
+
+   On Unix-like systems (add to your shell configuration file, e.g., `.bashrc` or `.zshrc`):
+   ```bash
+   alias rstring="python /path/to/rstring/directory/rstring/__main__.py"
```

Comment:
What happens if `rstring` has already been installed with pipx? Should we choose a different name to avoid a collision (e.g. `rstring-dev`)?

Reply:
Good point, updated